src/diffusers/pipelines/wuerstchen3/modeling_wuerstchen3_diffnext.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2023 Dominic Rampas MIT License
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import torch
+import torch.nn as nn
+
+from ...configuration_utils import register_to_config
+from .modeling_wuerstchen3_common import WuerstchenV3Unet
+
+
+class WuerstchenV3DiffNeXt(WuerstchenV3Unet):
```

Comment:
No abstraction from other models, this goes fundamentally against the design of the library